tests/search.spec.js

```diff
@@ -84,12 +84,22 @@ test("After search click profile", async ({ page }) => {
   await expect(profileHeaderText).toContain("Eddie Jaoude"); 
 });
 
-test.fixme(
-  "find the profile after providing concise name",
-  async () => {
-    // 1. click on search profile
-    // 2. type the whole name
-    // 3. Display the profile if the name is correct
+test("find the profile after providing concise name", async ({page}) => {
+  // 1. Start from the homepage
+  await page.goto("/");
+
+  // 2. look for and click on the search element   
+  const searchLink = page.locator("ul.ml-10.flex.items-baseline.space-x-4 li:first-child a[aria-label='Search'][href='/search']");
```

Comment:
Does the selector need to be so specific? I think it will make it fragile, can we use ?

```js
const profileLinkSelector = 'a[href="/eddiejaoude"]';
```

Reply:
Hi Eddie, to be honest, this wasn't my first choice. I should have mentioned in the request that I tested several diferent selector combinations, but all of the general selectors such as `'a[href="/search"]'` end up yielding 3 or more results, which rendered the selection ineffective. 

I understand your concern as the current version is dependent on the nested structure and tailwind styling, which could change over time. 

I was just testing out some other selector and the following one worked as well. What do you think?
```javascript 
"nav ul:first-child > li:first-child > a[href='/search']"
```
In the future, I'd suggest using ids in the a tags of all the navigation elements for easier and streamlined selection.

Reply:
<img width="1119" alt="Screenshot 2023-09-16 at 11 22 46" src="https://github.com/EddieHubCommunity/BioDrop/assets/113840653/b2dc0937-3b7b-4c86-a1fe-a0cda4417918">

Reply:
<img width="1113" alt="Screenshot 2023-09-16 at 11 23 44" src="https://github.com/EddieHubCommunity/BioDrop/assets/113840653/e5c91ee7-9ee2-4464-bfa6-507da233f661">